app/tsconfig.json

```diff
@@ -1,16 +1,17 @@
 {
   "compilerOptions": {
-    "target": "ES2023",
+    "target": "es5",
```

Comment:
I'm not sure why these changes are necessary? Can you please remove them, or let me know why they are needed? This code is running in electron with a recent javascript... so I'm unclear why es5 code should be generated.